app/assets/javascripts/classes/MetricNodeVM.ts

```diff
@@ -22,15 +22,17 @@ import ko = require('knockout');
 import ns = require('naturalSort');
 
 class MetricNodeVM implements BrowseNode {
+    metricName: KnockoutObservable<string>;
     name: KnockoutObservable<string>;
     children: KnockoutObservableArray<BrowseNode>;
     subfolders: KnockoutObservableArray<BrowseNode>;
     expanded: KnockoutObservable<boolean>;
     renderAs: KnockoutObservable<string>;
     visible: KnockoutObservable<boolean>;
 
-    constructor(name: string, id: string) {
-        this.name = ko.observable(name);
+    constructor(metricName: string, displayName: string, id: string) {
+        this.metricName = ko.observable(metricName);
```

Comment:
Why do I feel that we had this duality before and explicitly removed it?

Is it possible to do this with the existing name attribute? Obviously it will need to be formatted for display.